examples/http/opnsense_interfaces_ipv4.json

```diff
@@ -0,0 +1,42 @@
+{
+    "logo": "opnsense.svg",
+    "version": "1.0.0",
+    "name": "OPNSense Interfaces Stats IPV4",
+    "description": "Retrieves IPv4 interface statistics from an OPNsense firewall",
+    "category": "Software Integration",
+    "requirements": {
+        "credentials": true,
+        "sandbox_version": "1.12",
+        "others": [
+            "OPNsense",
+            "Firewall"
+        ]
+    },
+    "tags": [
+        "opnsense",
+        "firewall",
+        "interfaces",
+        "ipv4"
+    ],
+    "sample_period_s": 1800,
+    "has_actions": true,
+    "expected_variables": {
+        "table": "[6,999]"
```

Comment:
change to [6] 

(we've changed the way we measure them so each column is a single consumed variable)